src/abstractdataframe/abstractdataframe.jl

```diff
@@ -404,32 +392,33 @@ function StatsBase.describe(io, df::AbstractDataFrame)
         println(io, )
     end
 end
-StatsBase.describe(dv::AbstractArray) = describe(STDOUT, dv)
-function StatsBase.describe{T<:Number}(io, dv::AbstractArray{T})
-    if all(isna(dv))
+StatsBase.describe(nv::AbstractArray) = describe(STDOUT, nv)
+function StatsBase.describe{T<:Number}(io, nv::AbstractArray{T})
+    if all(_isnull, nv)
         println(io, " * All NA * ")
```

Comment:
Good catch. I should run a series of `git grep` to detect remnants of DataArrays. I haven't ported the documentation and benchmarks either.